Tests/Marketplace/marketplace_services.py

```diff
@@ -3765,3 +3774,33 @@ def get_id_set_entity_by_path(entity_path: Path, pack_folder: str, id_set: dict)
             if list(id_set_entity.values())[0]['file_path'] == str(entity_path):
                 return id_set_entity
     return {}
+
+
+def get_id_set_entity_by_display_name(display_name: str, pack_folder: str, id_set: dict):
+    """
+    Get the full entity dict from the id set of the entity given it's display name, if it does not exist in the id set
+    return None.
+
+    Args:
+        display_name: The display name of the entity (content item)
+        pack_folder: containing folder of that item
+        id_set: id set dict
+
+    Returns:
+        id set dict entity if exists, otherwise None
+    """
+    logging.debug(f"Checking if the entity with the display name {display_name} is present in the id set")
+
+    if not id_set:
+        return None
+
+    for id_set_entity in id_set[PACK_FOLDERS_TO_ID_SET_KEYS[pack_folder]]:
+
+        if list(id_set_entity.values())[0]['display_name'] == display_name:
+            return id_set_entity
+
+    if pack_folder == PackFolders.CLASSIFIERS.value:  # For Classifiers, check also in Mappers
```

Comment:
Are all classifiers appear under Mappers in id_set? I am sure we have a translation to this somewhere, no @yaakovi ?